src/lib/versionControl.ts

```diff
@@ -0,0 +1,37 @@
+interface Commit {
+    id: number;
+    message: string;
+    timestamp: Date;
+    content: string;
+  }
+  
+  export class VersionControl {
+    private commits: Commit[] = [];
+  
+    constructor(private filePath: string) {}
```

Comment:
What's `filePath` for?

Reply:
I think that was for the where to retreive the initial json file that gets committed upon.  library.json gets opened into  cards and then becomes the initial commit. then any changes on the cards is the following comitts